ykrt/src/compile/jitc_yk/codegen/x64/mod.rs

```diff
@@ -1745,7 +1745,10 @@ impl<'a> Assemble<'a> {
         let tgt_vars = tgt_vars.unwrap_or(self.loop_start_locs.as_slice());
         for (i, op) in self.m.loop_jump_vars().iter().enumerate() {
             let (iidx, src) = match op {
-                Operand::Var(iidx) => (*iidx, self.ra.var_location(*iidx)),
+                Operand::Var(iidx) => {
```

Comment:
Is this testable?

Reply:
It will get tested in an upcoming PR, but for boring reasons it's a bit awkward to do here: it's just a flat out bug that happens to cause problems in yklua (but you need several things to come together for it to explode).